WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -158,22 +158,28 @@ templates.class.WARRIOR = {
       args = {
         { spell = 100, type = "ability", requiresTarget = true, talent = {5,6}}, -- Charge
         { spell = 100, type = "ability", charges = true, requiresTarget = true, talent = 4, titleSuffix=" (2 Charges)"}, -- Charge
+        { spell = 121, type = "ability", requiresTarget = true}, -- Slam
```

Comment:
Unknown spell id: 121 and 129.

Reply:
oops! I typed macro id there, it's fixed, and i also renamed Cooldowns to Abilities as you said on discord